app/screens/channel/channel_base.js

```diff
@@ -107,6 +108,7 @@ export default class ChannelBase extends PureComponent {
 
         if (this.props.isSupportedServerCalls) {
             this.props.actions.loadCalls();
+            this.props.actions.loadConfig();
```

Comment:
crazy idea, but I saw something similar on the WS reconnection, perhaps we can create one action that execute this two things and batches the dispatch ? not sure if it is needed or wanted

Reply:
Yep, had to change up a bit, but it works. :)